e2e/app/element/clr-icon-link.ts

```diff
@@ -1,28 +1,21 @@
 import {Page, WaitForSelectorOptions} from 'puppeteer';
 import BaseElement from './base-element';
+import Container from './container';
 import TextOptions from './text-options';
-import {findIcon} from './xpath-finder';
+import * as xpathDefaults from './xpath-defaults';
```

Comment:
I know in Java we don't like to import `*` from a package. Not sure what the rule here is.